### ISSUE TITLE
[fried PI] Andrea's fried PI

### ISSUE THREAD

**Issue by AndreaCensi** [duckietown/docs-opmanual_duckiebot#36]
### Problem

Duckiebot does not boot.

No HDMI output.

Steady red light. 

### Events 

I was testing various Docker containers.

The Hat was on but never used to drive motors.

The 

### System information

What hardware was involved?

- [x] Raspberry PI 3B+
- [ ] Raspberry PI 3B
- [x] Motor hat version DB18
- [ ] Motor hat version DB17 (Adafruit)
- [x] PI Camera
- [x] HDMI and monitor
- [ ] LAN cable
- [ ] Other USB peripherals (joystick, keyboard, etc.)

*How was the raspberry Pi powered?*

- [x] Battery version DB18 (TODO: name?)
- [ ] Battery version DB17 (TODO: name?)
- [x] Power adapter
- [ ] Other batteries; specify: ___
- [ ] Powered using USB from a computer.

Sometimes I had only the PI connected to the battery but not the hat.

The PI reported under-voltage and throttling.

### Diagnostics

None - cannot access.

**Comment by MarcusAaltonen** [comment]
Fault search: 
1. No visible damaged can be detected on the Pi or HAT
2. The SD card was tested on another Pi and it booted up 
3. The HUT was removed and put into the factory tester where the motors turned and and LEDs turned on
4. The Pi was examined where it was observed that the voltage regulator generating the 3,3V 1,8V and 1,2V was not working. 
5. The fuse is intact and the regulator MXL7704 is getting 5V  

Tests:
1. Low input voltage:
I was able keep the Pi alive with an input voltage as low as 3.5V I tested using 2 different types of SD cards without damaging the Pi or SD card.

2. Low voltage with 100% CPU load:
I used Andreas SD card which was working together with the original HUT. 
I started the Pi and with 25% CPU load increments i got the following result:

input voltage | throttled | freq.       | temp from pi | CPU temp using thermocouple | time   | cpu load |  
                   4 | 0x50005 | 600MHz |                    42|                                               30 | 10:30 | 0
                   4 | 0x50005 | 600MHz |                    42|                                                38| 11:00 | 0
                   4 | 0x50005 | 600MHz |                    44|                                            40,6 | 11:11 | 25%
                   4 | 0x50005 | 600MHz |                47,2 |                                            43,1 | 11:20 | 50%
                   4 | 0x50005 | 600MHz |                49,4 |                                            44,8 | 11:30 | 75%
                   4 | 0x50005 | 600MHz |                52,1 |                                            45,1 | 12:30 | 100%
What we can see from the table above is that i left the Pi at 100% load and we reach a max temp of 52,1 and 45,1 degrees. after the test the Pi was rebooted and the same test was performed again with the same result. 

I now changed the input voltage to 5V and put it at 100% immediately:
input voltage | throttled | freq.       | temp from pi | CPU temp using thermocouple | time   | cpu load |  
                   5 |          0x0 | 600MHz |               41,2 |                                            41,9 | 14:45 | 0%
                   5 |          0x0 | 1200MHz |             69,8 |                                            66,7 | 15:45 | 100%
What we can see is that the unit never throttled. 

3. Short 5V to 3.3V 
Input voltage was now set to 5V, I had the Pi at 100% CPU load 
I shorted pin 1 and 2 (3.3V and 5V) and the HDMI stop immediately and the current dropped to 0.
All voltages on the regulator are now 0, the HUT and the SD was intact. This was repeated again with the same result. 

Result:  
I was able to reproduce the condition of the faulty raspberry Pi by shorting the 3.3V and 5V. 

Conclusion: 
By applying a low input voltage and loading the CPU at 100% neither the SD card, CPU or HUT was damaged. When having sufficient voltage I was not able to throttled the CPU even without a heatsink, future test has to be done to see if we can reach the teamp when the Pi start to throttled. 

Action points:
1. Change the screw from metal to plastic that is located just above the 5V and 3.3V pins.
2. Add in the instructions that ppl do a visual inspection of the HUT before applying it on the Pi, the pins bend easy.
3. Add a plastic protection for the pins  to make sure they cant short out one another.

**Comment by AndreaCensi** [comment]
@MarcusAaltonen am I correct to understand that there was no permanent damage?

**Comment by MarcusAaltonen** [comment]
No, the voltage regulator gets permanently damaged when I short 3.3V and 5V. 
If the regulator is damaged the whole Pi is scrap, that circuit is hard to repair and I would not trust a CPU that has been exposed to high voltage.

**Comment by AndreaCensi** [comment]
@MarcusAaltonen oh, so you used a *different* PI for the test?

**Comment by MarcusAaltonen** [comment]
I did the test two times on two different Pis with the same result, the voltage regulator stopped operating (just like the Pi you gave me )

**Comment by liampaull** [comment]
I don't understand the conclusion? Do we have some understanding of what's causing the destruction?

**Comment by tanij** [comment]
Conclusion: If we short the 5V and 3.3V pins we reproducibly get the same problems observed on Andrea's PI, i.e. a voltage regulator failing. This causes the other symptoms. 

 @liampaull are your "fried" pi's fried in the same way as the one andrea reported?

**Comment by MarcusAaltonen** [comment]
Hi ronie19, 
could you open up a new issue report and answer all the provided questions.  
https://github.com/duckietown/docs-opmanual_duckiebot/issues/new/choose 
Many thanks,
Marcus
